scripts/debug_tools/renew_info_files.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+# --------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# --------------------------------------------------------------------------
+
+import argparse
+import io
+import json
+import os
+import sys
+
+# Tried to import these from libcodechecker with no success. Pls help.
```

Comment:
One option is to add the `libcodechecker` directory to `PYTHONPATH` environment variable or use the `sys.path.append()` to add it from inside of the script, in that case this script needs to be kept relative to the `libcodechecker` module.
If not a lot of source code is copied or needed from `libcodechecker` it is ok to duplicate it here. `libcodechecker` is a quite big dependency with a lot of stuff not needed for this script.

Reply:
Thanks. I left the functions copied here then.